src/common/pages/cookie-policy/index.tsx

```diff
@@ -0,0 +1,77 @@
+import Layout from '@theme/Layout';
+import React, { useEffect } from 'react';
+
+export default function CookiePolicy() {
+
+
+  // Why is this needed?
+  // Injecting CookieBot as a global script in the HTML file 
+  // would make it load when the page loads instead of when the component is mounted
+  // And this would make it error because it would not be able to find where to inject the declaration
+  // So, we are injecting it twice, and just to be sure, we delete any extra cookie declaration, 
+  // as we only want to show one
+  useEffect(() => {
+
+    const appendCookiebotCdReport = new Promise((resolve, reject) => {
+      const script = document.createElement('script');
+      document.body.appendChild(script);
+      script.onload = resolve;
+      script.onerror = reject;
+      script.async = true;
+      script.src =
+        'https://consent.cookiebot.com/8f051d60-4ecb-41a0-abb9-4874fd999e4f/cdreport.js?referer=iota-wiki-fawn.vercel.app';
```

Comment:
before merging, this line has to be updated to the production URL 🙏🏼

Reply:
We also need to change back the side id to `13` [here](https://github.com/iota-wiki/iota-wiki/blob/326a62a09026b5747c51a228dcc90d82f057095a/static/js/matomo.js#L9)